src/gui/search/searchwidget.cpp

```diff
@@ -121,27 +121,43 @@ SearchWidget::SearchWidget(MainWindow *mainWindow)
     fillPluginComboBox();
 
     connect(m_searchPattern, SIGNAL(textEdited(QString)), this, SLOT(searchTextEdited(QString)));
-    connect(selectPlugin, SIGNAL(currentIndexChanged(const QString &)), this, SLOT(selectMultipleBox(const QString &)));
+    connect(selectPlugin, SIGNAL(currentIndexChanged(int)), this, SLOT(selectMultipleBox(int)));
 }
 
 void SearchWidget::fillCatCombobox()
 {
     comboCategory->clear();
     comboCategory->addItem(SearchEngine::categoryFullName("all"), QVariant("all"));
-    foreach (QString cat, m_searchEngine->supportedCategories()) {
-        qDebug("Supported category: %s", qPrintable(cat));
-        comboCategory->addItem(SearchEngine::categoryFullName(cat), QVariant(cat));
+    comboCategory->insertSeparator(1);
+
+    using QStrPair = QPair<QString, QString>;
+    QList<QStrPair> tmpList;
+    foreach (const QString &cat, m_searchEngine->supportedCategories())
+        tmpList << qMakePair(SearchEngine::categoryFullName(cat), cat);
+    qSort(tmpList.begin(), tmpList.end(), [](const QStrPair &l, const QStrPair &r) { return (l.first < r.first); } );
```

Comment:
replied in https://github.com/qbittorrent/qBittorrent/pull/4995/files/204c1814c899d66e7166c70d2618490631507169..8801eeffa68d636e8b1a3ab7a6f8a2282efb5f2d#r59157255